templates/taxi/car_detail.html

```diff
@@ -1,8 +1,14 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>Car detail:</h1>
+  <p>Model: {{ car.model }}</p>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <a href="{% url "taxi:car-update" pk=car.id %} " class="btn btn-primary">UPDATE</a>
```

Comment:
There is an extra space before the closing quotation mark in the `href` attribute of the update link. This might cause issues with URL resolution.